### ISSUE TITLE
Minor warnings related to tidyr > 1.0.0 compatibility 

### ISSUE THREAD

**Issue by hawkrobe** [langcog/tidyboot#9]
After `tidyr` version 1.0.0, this warning is thrown:

```
`cols` is now required.
Please use `cols = c(strap)`
```

There is also now a `tibble` deprecation warning:

```
`as_data_frame()` is deprecated as of tibble 2.0.0.
Please use `as_tibble()` instead.
The signature and semantics have changed, see `?as_tibble`.
```

**Comment by cjhook** [comment]
It looks like the 'cols is now required' warning is from unnest():

https://tidyr.tidyverse.org/reference/nest_legacy.html

**Comment by hawkrobe** [comment]
see also the profiling in #8 (where the updated `unnest()` is also the source of the slowdown).

**Comment by dyurovsky** [comment]
fixed by 082557d4599607e458b876bdd48351b98376378c
